Source/towners.cpp

```diff
@@ -414,7 +414,7 @@ void InitFarmer()
 	}
 	towners[numtowners]._tNFrames = 15;
 	NewTownerAnim(numtowners, towners[numtowners]._tNAnim[DIR_S], towners[numtowners]._tNFrames, 3);
-	strcpy(towners[numtowners]._tName, _("Lester the farmer"));
+	towners[numtowners]._tName = {_("Lester the farmer")};

```

Comment:
`string_view` is implicitly convertible from `const char *`, so the curly braces aren't necessary (throughout)